phastSim/phastSim.py

```diff
@@ -753,9 +753,10 @@ def init_insertion_frequencies(self, ref):
 
 
 class GenomeTree_hierarchical:
-    def __init__(self, nCodons, codon, ref, gammaRates, omegas, mutMatrix, hyperCategories, hyperMutRates, 
-                indels, insertionRate, insertionLength, insertionFrequencies, deletionRate, deletionLength, scale, infoFile, verbose, noNorm, mutationsTSVinput):
+    def __init__(self, nCodons, codon, ref, gammaRates, omegas, mutMatrix, hyperCategories, hyperMutRates,
+                indels, insertionRate, insertionLength, insertionFrequencies, deletionRate, deletionLength, scale, infoFile, verbose, noNorm, mutationsTSVinput, args):
 
+        self.args = args
```

Comment:
also don't keep these two changes

Reply:
Also done in [008bbce](https://github.com/NicolaDM/phastSim/pull/20/commits/008bbce8fe409c1e636ff780a70ed90b8e5edc57)